### PR TITLE
Fix gazebo scene viewer build

### PR DESCRIPTION
**Pull request by iche033**
## Summary

Fixes compile errors:
* xlib and protobuf have conflicting `Status` definition
* there is `gazebo::msgs::Capsule` msg type so commented out code for creating capsules.

**Note** The demo segfaults at run time. This does not happen in ign-rendering4. Ticketed https://github.com/ignitionrobotics/ign-rendering/issues/290

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge**

### CODE REVIEW

File: examples/gazebo_scene_viewer/SceneManager.cc [hunk 1]
```diff
@@ -1509,13 +1510,14 @@ void SubSceneManager::ProcessCone(
...
     const gazebo::msgs::Geometry & _geometryMsg, VisualPtr _parent)
 {
-  GeometryPtr capsule = this->activeScene->CreateCapsule();
-  const gazebo::msgs::CapsuleGeom &capsuleMsg = _geometryMsg.capsule();
-  double x = 2 * capsuleMsg.radius();
-  double y = 2 * capsuleMsg.radius();
-  double z = capsuleMsg.length();
-  _parent->SetLocalScale(x, y, z);
-  _parent->AddGeometry(capsule);
+  // \todo(anyone) needs gazebo capsule msg
```
Comment (ahcorde) [inline examples/gazebo_scene_viewer/SceneManager.cc:1513]: This package should use `ign-msgs7` and the message it's there https://github.com/ignitionrobotics/ign-msgs/blob/main/proto/ignition/msgs/geometry.proto

File: examples/gazebo_scene_viewer/SceneManager.cc [hunk 2]
```diff
@@ -1833,12 +1835,13 @@ void SubSceneManager::CreateGeometryFunctionMap()
   this->geomFunctions[gazebo::msgs::Geometry::BOX] =
       &SubSceneManager::ProcessBox;
 
-  // TODO(anyone): enable when cone protobuf msg created
+  // todo(anyone): enable when cone protobuf msg is created
```
Comment (ahcorde) [inline examples/gazebo_scene_viewer/SceneManager.cc:1838]: The Cone msg is there too https://github.com/ignitionrobotics/ign-msgs/blob/main/proto/ignition/msgs/geometry.proto#L74

**Review by ahcorde — CHANGES_REQUESTED** [review]

File: examples/gazebo_scene_viewer/SceneManager.cc (hunk 1 above)
Comment (iche033) [reply examples/gazebo_scene_viewer/SceneManager.cc:1513]: the errors are actually related to `gazebo::msgs` (we have these msgs in ignition but not in gazebo). I think we would need to add the cone and capsule msgs in gazebo: https://github.com/osrf/gazebo/tree/gazebo11/gazebo/msgs

**Review by ahcorde — APPROVED** [review]

File: examples/gazebo_scene_viewer/SceneManager.cc (hunk 1 above)
Comment (ahcorde) [reply examples/gazebo_scene_viewer/SceneManager.cc:1513]: I created an issue https://github.com/ignitionrobotics/ign-rendering/issues/293

**Review by chapulina — APPROVED** [review]

**Pull request merged**